contract_section/tests/test_contract.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2016 Tecnativa - Carlos Dauden
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# Copyright 2018 Roel Adriaans - roel@road-support.nl
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.contract.tests.test_contract import TestContractBase
+
+
+class TestContract(TestContractBase):
+    @classmethod
+    def setUpClass(cls):
+        # Reuse the tests from the contract module, but replace the
+        # contract line
+        super(TestContract, cls).setUpClass()
+        cls.acct_line.unlink()
```

Comment:
Isn't easier to just do:

```python3
cls.acct_line.layout_category_id = cls.section.id
cls.line_vals['layout_category_id'] = cls.section.id
```
instead of unlinking and creating again?

Reply:
I rather test a clean line..
Test result should be the same, and isn't explicit is better than implicit? :)

Reply:
Well, I think it's also explicit enough as you have the line created in super, but I won't oppose on this as you have already done a good job.